lib/const.rb

```diff
@@ -61,5 +61,5 @@
   CREW_BUILD = 'x86_64-cros-linux-gnu'
 end
 CREW_OPTIONS = "--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --mandir=#{CREW_MAN_PREFIX} --build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD}"
-CREW_MESON_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -DLIB_INSTALL_DIR=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -DSYSCONFDIR=#{CREW_PREFIX}/etc -Ddatadir=#{CREW_LIB_PREFIX} -Dbuildtype=release"
```

Comment:
Bump crew version?  How about `CREW_CMAKE_OPTIONS`?

Reply:
I didn't want to touch CMAKE_OPTIONS until MESON was taken care of.

Reply:
But yes I want to clean CREW_CMAKE_OPTIONS up  too by taking out ```-DSYSCONFDIR=#{CREW_PREFIX}/etc -Ddatadir=#{CREW_LIB_PREFIX}```

Reply:
The question remaining is: Does ```--build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD}"``` work with cmake?